src/styles/index.scss

```
@@ -1,3 +1,82 @@
+$sectionSize: 100px;
+$gapBetweenSections: 1px;
+$startDays: (
+  mon: 0,
+  tue: 1,
+  wed: 2,
+  thu: 3,
+  fri: 4,
+  sat: 5,
+  sun: 6,
+);
+
+* {
+  box-sizing: border-box;
+}
+
 body {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
   margin: 0;
+
+  .calendar {
+    display: flex;
+    flex-wrap: wrap;
+    gap: $gapBetweenSections;
+    padding: 10px;
+    width: calc(($sectionSize * 7) + 20px + ($gapBetweenSections * 6));
+
+    &__day {
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      transition-duration: 0.5s;
+    }
+
+    @for $day from 1 through 31 {
+      &__day:nth-child(#{$day}) {
+        border: 1px solid black;
+        background-color: #eee;
+        width: $sectionSize;
+        height: $sectionSize;
+
+        &::before {
+          content: '#{$day}';
+          display: block;
+          text-align: center;
+          width: $sectionSize;
+          line-height: 100px;
+          font-size: 30px;
+          font-family: Arial, Helvetica, sans-serif;
+        }
+      }
```

```suggestion
        &::before {
          content: '#{$day}';
        }
      }
```
Consider using general class for common styles instead of generating them in the cycle

---

`body {
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  font-size: 30px;
  font-family: Arial, Helvetica, sans-serif;
  height: 100vh;
  margin: 0;
  
      &__day {
      display: flex;
      justify-content: center;
      align-items: center;
      transition-duration: 0.5s;
      border: 1px solid black;
      background-color: #eee;
      width: $sectionSize;
      height: $sectionSize;
    }

    @for $day from 1 through 31 {
      &__day:nth-child(#{$day}):before {
          content: '#{$day}';
        }
      }`

Does it seem right?